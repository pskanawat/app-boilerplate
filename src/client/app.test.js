import App from "./app";
import React from 'react';
import {shallow} from 'enzyme';

describe("App", () => {
	test("Should match the text node", () => {
		const wrapper = shallow(<App />);
		expect(wrapper.find("div").text()).toBe("Welcome to React JS App");
		expect(wrapper).toMatchSnapshot();
	});
});