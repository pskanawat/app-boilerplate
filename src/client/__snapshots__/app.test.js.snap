// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should match the text node 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Welcome to React JS App",
    },
    "ref": null,
    "rendered": "Welcome to React JS App",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Welcome to React JS App",
      },
      "ref": null,
      "rendered": "Welcome to React JS App",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
